src/sentry/event_manager.py

```diff
@@ -777,21 +777,8 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         # save the event unless its been sampled
         if not is_sample:
-            try:
-                with transaction.atomic(using=router.db_for_write(Event)):
-                    event.save()
-            except IntegrityError:
```

Comment:
Is this still a valid case ?
I suspect this can happen in case of race conditions if this method is executed twice concurrently.
What happens if we try to write the events twice on nodestore ?

Reply:
I don't think it'll raise an exception. Last one will win.